blog/2023-08-08-on-lab-integration/index.md

```diff
@@ -0,0 +1,51 @@
+---
+slug: 2023/08/on-lab-integration
+title: On Lab Integration
+authors: claudemamo
+tags: [lab-integration, lis, lab-machines]
+---
+
+COVID-19 has highlighted the importance of having public laboratories efficiently integrated with the rest of the healthcare architecture. It’s been shown through numerous case studies that connecting labs with the country’s health information exchanges (HIEs) significantly facilitates critical activities such as disease surveillance, notification of test results, reporting to stakeholders, and inventory control. It follows that DHIS2 can play a pivotal role in these use cases. Imagine the national Health Management Information System (HMIS) displaying the daily tally of new positive tuberculosis cases on its dashboard as soon as the test results are available at the lab, or a surveillance system notifying patients of their results. Another innovative use case gaining traction is for DHIS2 to collect the state of lab diagnostics (i.e., error rates, usage frequency, online status, etc…) in order to simplify their management.
+
+_A typical public lab (source: https://drdollah.com/laboratory-information-system-lis/)_
+
+![A Lab](./lab-flow1.png)
+
+HISP Centre, for the past year or so, has been exploring the lab integration space to understand what’s been done so far, the challenges, and how it can be improved. A major hurdle we identified to seamlessly integrate labs with health systems is that labs around the globe have technicians, receptionists, and other personnel manually punching in data, copying and filtering spreadsheet rows, mapping test requests and results, and double-checking data for quality control. This is especially true for low and middle-income countries (LMICs). In the initial COVID-19 testing response, such work practices couldn’t keep up with the volume of test requests and turnaround time suffered as a result. A number of countries couldn’t allow this state of affairs to continue therefore initiatives were created to partially or fully digitise their lab workflows. We’ve seen success stories in these initiatives. In Rwanda, [for example](https://dhis2.org/rwanda-covid-lab-integration/), [HISP Rwanda](https://hisprwanda.org/) developed an application for the national reference lab that validates COVID-19 test orders at the reception stage followed by validation of test results prior to transmitting these results to Rwanda's DHIS2 Tracker COVID-19 surveillance instance. The application didn’t fully manage the workflow (e.g., spreadsheets were still shared out-of-band) as witnessed during our on-site visit but enough of the workflow was digitised to substantially reduce test turnaround time.
```

Comment:
```suggestion
HISP Centre, for the past year or so, has been exploring the lab integration space to understand what’s been done so far, the challenges, and how it can be improved. A major hurdle we identified to seamlessly integrate labs with health systems is that labs around the globe have technicians, receptionists, and other personnel manually punching in data, copying and filtering spreadsheet rows, mapping test requests and results, and double-checking data for quality control. This is especially true for low and middle-income countries (LMICs). In the initial COVID-19 testing response, such work practices couldn’t keep up with the volume of test requests and turnaround time suffered as a result. 

A number of countries couldn’t allow this state of affairs to continue therefore initiatives were created to partially or fully digitise their lab workflows. We’ve seen success stories in these initiatives. In Rwanda, [for example](https://dhis2.org/rwanda-covid-lab-integration/), [HISP Rwanda](https://hisprwanda.org/) developed an application for the national reference lab that validates COVID-19 test orders at the reception stage followed by validation of test results prior to transmitting these results to Rwanda's DHIS2 Tracker COVID-19 surveillance instance. The application didn’t fully manage the workflow (e.g., spreadsheets were still shared out-of-band), as witnessed during our on-site visit, but enough of the workflow was digitised to substantially reduce test turnaround time.
```
Added 2 comma's in last sentence